src/eureka/S3_data_reduction/background.py

```diff
@@ -259,13 +261,17 @@ def fitbg(dataim, meta, mask, x1, x2, deg=1, threshold=5, isrotate=False,
                     model = np.polyval(coeffs, goodxvals)
                     # Calculate residuals and number of sigma from the model
                     residuals = dataslice - model
-                    # Simple standard deviation (faster but prone to missing
-                    # scanned background stars)
-                    stdres = np.std(residuals)
-                    # Median Absolute Deviation (slower but more robust)
-                    # stdres  = np.median(np.abs(np.ediff1d(residuals)))
-                    # Mean Absolute Deviation (good compromise)
-                    # stdres = np.mean(np.abs(np.ediff1d(residuals)))
+                    #Choose method for finding bad pixels
+                    if hasattr(meta, 'bg_method') and meta.bg_method=='std':
```

Comment:
What happens if `bg_method` is not given?  We should default to using `std` or `mean`.